src/lambda_function/s3_event_config/template.yaml

```diff
@@ -45,6 +46,13 @@ Resources:
           S3_TO_GLUE_FUNCTION_ARN: !Ref S3ToGlueFunctionArn
           BUCKET_KEY_PREFIX: !Ref Namespace
 
+  LambdaInvokePermission:
+    Type: AWS::Lambda::Permission
+    Properties:
+      FunctionName: !Ref S3ToGlueFunctionArn
```

Comment:
Should this be invoking the Arn of `S3EventConfigFunction`?

Reply:
Yes.